t/27-consoles-s3270.t

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/perl
+
+use Data::Dumper;
```

Comment:
I guess you needed that only for debugging

Reply:
It'd still there...

Reply:
@gauravpathak it does not seem as if you use Data::Dumper in this test module. Please only keep module imports that are actually used